tools/dataset/parse_cofw_dataset.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+import time
+
+import cv2
+from hdf5storage import loadmat
```

Comment:
Can hdf5storage be replaced by h5py or scipy?

Reply:
`scipy.io` does not work for v7.3 or higher matlab.
I have replaced `hdf5storage` with `h5py`.